Apps/Src/ArrayConnection.c

```diff
@@ -1,4 +1,5 @@
 /* Copyright (c) 2020 UT Longhorn Racing Solar */
+// DEPRECATED
```

Comment:
Why did you comment stuff out instead of deleting it in this file?

Reply:
Deleted